docs/source/advanced_usage.rst

```diff
@@ -32,6 +32,84 @@ locally):
 - QVM URL: ``http://127.0.0.1:5000``
 - quilc URL: ``tcp://127.0.0.1:5555``
 
+Concurrency
```

Comment:
I found when testing locally for lots of programs/threads, I had to increase the timeout on `get_qc()` with `get_qc(..., timeout=...)`. The default timeout everywhere is 5 seconds. Is this reasonable, or should it go back to 10 seconds as it was in pyquil v2? I had updated it in v3 to align with `httpx`, but it's not super necessary. It's sort of a blanket timeout for all types of requests (getting engagements, compiling, running, etc).

Reply:
Also, does it sound reasonable that you might need to increase the timeout at all, or is it a smell of a problem? If it's reasonable, I'll add a note about it to this section of the docs.

It sounds reasonable to me, given that programs are still being queued and run sequentially in lodgepole. If you have more programs ahead of you in line, it could take longer to get your results (it's probably the `get_buffers` rpcq request that takes the most time).

Reply:
I was surprised to see the timeout reduced. If anything, the timeout is going to need to be longer when folks start compiling against e.g. 100q devices.

Reply:
OK, I'll restore the default timeout everywhere, and I'll also document the timeout here for concurrent runs

Reply:
Done.